<!DOCTYPE html>
<!--
 | Generated by Apache Maven Doxia at 2015-08-14 
 | Rendered using Apache Maven Fluido Skin 1.4
-->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta name="Date-Revision-yyyymmdd" content="20150814" />
    <meta http-equiv="Content-Language" content="en" />
    <title>Apache Taverna &#x2013; Disease-related research</title>
    <link rel="stylesheet" href="../../css/apache-maven-fluido-1.4.min.css" />
    <link rel="stylesheet" href="../../css/site.css" />
    <link rel="stylesheet" href="../../css/print.css" media="print" />

      
    <script type="text/javascript" src="../../js/apache-maven-fluido-1.4.min.js"></script>

    
    <meta name="Notice" content="Licensed to the Apache Software Foundation (ASF) under one            or more contributor license agreements.  See the NOTICE file            distributed with this work for additional information            regarding copyright ownership.  The ASF licenses this file            to you under the Apache License, Version 2.0 (the            &quot;License&quot;); you may not use this file except in compliance            with the License.  You may obtain a copy of the License at            .              http://www.apache.org/licenses/LICENSE-2.0            .            Unless required by applicable law or agreed to in writing,            software distributed under the License is distributed on an            &quot;AS IS&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY            KIND, either express or implied.  See the License for the            specific language governing permissions and limitations            under the License." />              </head>
        <body class="topBarEnabled">
          
                        
                    
                

    <div id="topbar" class="navbar navbar-fixed-top ">
      <div class="navbar-inner">
                                  <div class="container" style="width: 90%;"><div class="nav-collapse">
            
                
                                                                                <a class="brand" href="../../"  title="Apache Taverna">

                                
                                                                                                                    <img src="../../img/taverna-logo-145x134.png" alt="Apache Taverna" />
                
                </a>
                    
                                <ul class="nav">
                          <li class="dropdown">
        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Project <b class="caret"></b></a>
        <ul class="dropdown-menu">
        
                      <li>      <a href="../../index.html"  title="Home">Home</a>
</li>
                  
                      <li>      <a href="../../community/"  title="Community">Community</a>
</li>
                  
                      <li>      <a href="../../documentation/"  title="Documentation">Documentation</a>
</li>
                  
                      <li>      <a href="../../download/"  title="Download">Download</a>
</li>
                  
                      <li>      <a href="../../about/"  title="About us">About us</a>
</li>
                          </ul>
      </li>
                <li class="dropdown">
        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Introduction <b class="caret"></b></a>
        <ul class="dropdown-menu">
        
                      <li>      <a href="../../introduction/"  title="Introduction to Taverna">Introduction to Taverna</a>
</li>
                  
                      <li>      <a href="../../introduction/features"  title="Features">Features</a>
</li>
                  
                      <li>      <a href="../../introduction/why-use-workflows"  title="Why use workflows?">Why use workflows?</a>
</li>
                  
                      <li>      <a href="../../#"  title="... (TODO)">... (TODO)</a>
</li>
                          </ul>
      </li>
                <li class="dropdown">
        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Documentation <b class="caret"></b></a>
        <ul class="dropdown-menu">
        
                      <li>      <a href="../../documentation/"  title="Taverna docs">Taverna docs</a>
</li>
                  
                      <li>      <a href="../../documentation/glossary/"  title="Glossary">Glossary</a>
</li>
                  
                      <li>      <a href="../../documentation/scufl2/"  title="SCUFL2">SCUFL2</a>
</li>
                  
                      <li>      <a href="../../#"  title="... (TODO)">... (TODO)</a>
</li>
                          </ul>
      </li>
                <li class="dropdown">
        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Download <b class="caret"></b></a>
        <ul class="dropdown-menu">
        
                      <li>      <a href="../../download/"  title="Which Taverna to download?">Which Taverna to download?</a>
</li>
                  
                      <li>      <a href="../../download/language/"  title="Language">Language</a>
</li>
                  
                      <li>      <a href="../../download/workbench/"  title="Workbench">Workbench</a>
</li>
                  
                      <li>      <a href="../../download/command-line-tool/"  title="Command Line Tool">Command Line Tool</a>
</li>
                  
                      <li>      <a href="../../#"  title="... (TODO)">... (TODO)</a>
</li>
                          </ul>
      </li>
                <li class="dropdown">
        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Community <b class="caret"></b></a>
        <ul class="dropdown-menu">
        
                      <li>      <a href="../../community/"  title="Taverna Community">Taverna Community</a>
</li>
                  
                      <li>      <a href="../../community/lists"  title="Mailing lists">Mailing lists</a>
</li>
                  
                      <li>      <a href="../../community/contribute"  title="Contribute">Contribute</a>
</li>
                  
                      <li>      <a href="../../#"  title="... (TODO)">... (TODO)</a>
</li>
                          </ul>
      </li>
                  </ul>
          
                      <form id="search-form" action="https://www.google.com/search" method="get"  class="navbar-search pull-right" >
    
  <input value="$sitesearchValue" name="sitesearch" type="hidden"/>
  <input class="search-query" name="q" id="query" type="text" />
</form>
<script type="text/javascript" src="https://www.google.com/coop/cse/brand?form=search-form"></script>
          
                                                                            
                               <ul class="nav pull-right">
              <li class="dropdown">
                <a href="#" class="dropdown-toggle" data-toggle="dropdown">External Links <b class="caret"></b></a>
                <ul class="dropdown-menu">
                      <li>      <a href="https://apache.org/"  title="Apache Software Foundation">Apache Software Foundation</a>
</li>
      <li>      <a href="https://incubator.apache.org/"  title="Apache Incubator">Apache Incubator</a>
</li>
                  </ul>
              </li>
            </ul>
          
                      </div>
          
        </div>
      </div>
    </div>
    
        <div class="container">
          <div id="banner">
        <div class="pull-left">
                                                  <a href="../.././" id="bannerLeft">
                <h2>Apache Taverna</h2>
                </a>
                      </div>
        <div class="pull-right">  </div>
        <div class="clear"><hr/></div>
      </div>

      <div id="breadcrumbs">
        <ul class="breadcrumb">
                
                    
                  <li id="publishDate">Last Published: 2015-08-14
                   </li>
                      
                
                    
      
                            </ul>
      </div>

      
                
        <div id="bodyColumn" >
                                  
            
<p>Taverna is used by various projects and researchers for investigations into various diseases:</p>

<ul>
  
<li><a href="#vph-dareit">Dementia</a> (<a class="externalLink" href="mailto:VPH-DARE@IT">VPH-DARE@IT</a> and VPH-SHARE)</li>
  
<li><a href="#analysis-of-anthrax-bacterium">Analysis of the anthrax bacterium</a></li>
  
<li><a href="#trypanosomiasis">Study of resistance into trypanosomiasis</a> (sleeping sickness)</li>
  
<li><a href="#graves-disease">Identification of genes linked to Graves disease</a></li>
  
<li><a href="#williams-beuren-syndrome">Characterisation of genes associated with Williams-Beuren syndrome</a></li>
  
<li><a href="/introduction/related-projects#cagrid">Cancer research</a> (caGrid)</li>
</ul>
<p><a name="vph-dareit"></a></p>
<div class="section">
<h2><a name="VPH-DAREIT"></a><a class="externalLink" href="mailto:VPH-DARE@IT">VPH-DARE@IT</a></h2>
<p>The Center for Computational Imaging and Simulation Technologies in Biomedicine at the  <a class="externalLink" href="http://sheffield.ac.uk/">University of Sheffield</a>, is leading the &#x201c;Virtual Physiological Human:  DementiA Research Enabled by IT&#x201d; (<a class="externalLink" href="http://www.vph-dare.eu/">VPH-DARE@IT</a>) project,  which aims to provide a systematic,  multifactorial and multiscale modelling approach to understanding dementia onset and  progression and enable more objective, earlier,  predictive and individualised diagnoses and prognoses of dementias to cope with the challenge of an ageing European society.  See <a class="externalLink" href="http://www.vph-dare.eu/">http://www.vph-dare.eu/</a> for more details.</p>
<p>The project will use <a class="externalLink" href="http://vph-share.eu/">VPH-SHARE</a> infrastructure and  the GIMIAS software along with Taverna Workbench and Taverna Server technology for the execution and  composition of medical simulation workflows. These workflows will be focused on the analysis of medical data from thousands of individuals and populations;  as well as genetic, biochemical and metabolic pathway models. The results of the analysis aim to support highly innovative clinical decision systems for  fighting the colossal health and societal burden resulting from dementias.</p>
<p><a name="analysis-of-anthrax-bacterium"></a></p></div>
<div class="section">
<h2><a name="Analysis_of_anthrax_bacterium"></a>Analysis of anthrax bacterium</h2>
<p>Taverna has been used by Anil Wipat and others to automate a series of analyses on all the proteins  produced by a bacterium to create, by a process of selection and elimination,  a list of secreted proteins and their properties.</p>
<p>The secreted proteins explain why anthrax can grow in animal hosts but not in soil.</p>
<div class="section">
<h3><a name="Publications"></a>Publications</h3>
<p>The <a class="externalLink" href="http://www.allhands.org.uk/2006/proceedings/papers/709.pdf">paper</a>  <i>e-Science Tools For The Genomic Scale Characterisation Of Bacterial Secreted Proteins</i>  by Craddock et al. gives an overview of this work.</p>
<p><a name="trypanosomiasis"></a></p></div></div>
<div class="section">
<h2><a name="Trypanosomiasis_sleeping_sickness"></a>Trypanosomiasis (sleeping sickness)</h2>
<p>One of the major goals of biology, and consequently bioinformatics,  is to successfully bridge the gap between genotype and phenotype. Microarray and Quantitative Trait Loci data are increasingly used to aid in the discovery of candidate genes  that might be responsible for phenotypic differences. In previous years, studies into genotype-phenotype correlations have been conducted manually. This has led to problems with regards to the identification of functional candidate genes,  primarily due to the scale of data being investigated and the reliance on specific expertise,  which may bias the investigations outcome.</p>
<p>With the development and connection of Web services into workflows, however,  these large scale datasets can be processed systematically enabling detailed information to be gathered,  published and subsequently re-investigated,  enhancing the possibility of bridging the gap between genotype and phenotype using pathways.</p>
<p>So far this investigation has highlighted the issues facing the manual analysis of microarray and QTL data,  and how automated approaches provide a systematic means to investigate genotype-phenotype correlations. We were able to illustrate how the large scale analysis of microarray gene expression and quantitative trait data,  investigated at the level of biological pathways,  enables links between genotype and phenotype to be successfully established.</p>
<p>An example of a workflow used to gather pathway information for candidate genes from a QTL region is shown below. These workflows have so far been applied to two different genotype-phenotype problems:</p>

<ul>
  
<li>The study of resistance to African Trypanosomiasis in mice, infected with Trypanosoma congolen</li>
  
<li>The study of immunological effects and parasite expulsion in mice infected with Trichuris muris</li>
</ul>
<p>Using this systematic,  pathway driven approach, we were able to successfully identify a candidate gene and  biological pathway believed to be strongly associated with resistance to African Trypanosomiasis.</p>
<p>The workflow for performing this analysis is available on  <a class="externalLink" href="http://www.myexperiment.org/workflows/16">myExperiment</a>.</p>
<p><a class="externalLink" href="http://www.myexperiment.org/workflows/16"><img src="/img/trypanosomiasis.jpg" alt="pathways_and_gene_annotations_for_qtl_phenotype_28303_3" /></a></p>

<div class="section">
<h3><a name="Publications"></a>Publications</h3>
<p><a class="externalLink" href="http://scholar.google.co.uk/scholar?hl=en&amp;q=Taverna+trypanosomiasis">Articles and papers</a>  about the success of Taverna for trypanosomiasis (sleeping sickness) research.</p>
<p><a name="graves-disease"></a></p></div></div>
<div class="section">
<h2><a name="Graves_disease"></a>Graves disease</h2>
<div class="section">
<h3><a name="Graves_disease_scenario"></a>Graves disease scenario</h3>
<p>The aim of this scenario is to identify and characterise genes which are located in regions on human chromosomes  which show linkage to Graves disease (GD) (shown in figure below). GD is an autoimmune disease of the thyroid in which the immune system of an individual attacks cells  in the thyroid gland resulting in hyperthyroidism. This is caused by the stimulation of the thyrotrophin receptor by thyroid-stimulating autoantibodies secreted by  lymphocytes of the immune system.</p>
<p><a href="/img/gravesdiseasescenario_1.png">  <img src="/img/gravesdiseasescenario_1.png" title="gravesdiseasescenario_1" alt="Graves Disease Scenario" width="249" height="300" /></a></p></div>
<div class="section">
<h3><a name="Affymetrix_microarray_studies"></a>Affymetrix microarray studies</h3>
<p>The GD candidate genes were identified by microarray analysis. Affymetrix U95A arrays were probed with RNA extracted from CD4 positive lymphocytes from four GD patients  and four healthy controls. The four GD microarray datasets were then compared to the four control datasets using  the Affymetrix data mining tool to identify differentially expressed genes.</p></div>
<div class="section">
<h3><a name="Annotation_pipeline"></a>Annotation pipeline</h3>
<p>Over 50 genes were found to be differentially-expressed in CD4 positive lymphocytes from GD patients. In order to understand why these genes were expressed in lymphocytes from GD patients but not in healthy individuals,  the GD biologist would like to use myGrid to query public databases such as  EMBL, GO, HGVBASE and MEDLINE to view information about gene structure and function,  chromosome location, the presence of single nucleotide polymorphisms (SNPs),  expression control features and association with other genetic diseases. The experimental conditions and diseases in which the expression of the candidate genes  are significantly altered also need to be identified from OMIM.</p></div>
<div class="section">
<h3><a name="Genotype_assay_design_system"></a>Genotype assay design system</h3>
<p>SNPs are small (single base pair) changes genetic variations which are found in the genome amongst individuals. The differential expression of the candidate genes in GD individuals may be due or related to the presence of  SNPs associated with GD. The GD biologist is interested in identifying and determining the frequency of those SNPs which are found in her GD patients.</p>
<p>Restriction fragment length polymorphism (RFLP) assays are developed to genotype SNPs in her candidate genes. A region flanking either side of the SNP is amplified using polymerase chain reaction (PCR). The amplified PCR product is digested with a suitable restriction enzyme  (i.e. one that will cut at one SNP allele and not the other)  and the products are run on agarose gels to view product size and determine the genotype.</p>
<p>The GD biologist would like to use myGrid to:</p>

<ul>
  
<li>Query databases to retrieve SNP information associated with candidate genes.</li>
  
<li>Aid in the design of primers  (bits of DNA which signify the start and end points of the section of the DNA sequence which she wants to amplify)  for the PCR experiment.</li>
  
<li>Select the restriction enzyme that is specific to a particular SNP for the RFLP experiment.</li>
</ul></div>
<div class="section">
<h3><a name="a3D_protein_structure__effect_of_coding_SNP_on_protein_active_site"></a>3D protein structure &amp; effect of coding SNP on protein active site</h3>
<p>Any SNPs occurring in the coding regions of a candidate gene may potentially give rise to  a change in the amino acid sequence of the protein encoded by the gene. The GD biologist would like to use myGrid to:</p>

<ul>
  
<li>Query a protein structure database, e.g. PDB or MSD,  to determine whether a structure of the protein encoded by her candidate gene is available.  If so, view the protein structure to study how it relates to the function of the protein.</li>
  
<li>Obtain information about the protein, e.g. its function and functional domains, by querying SWISS-PROT and InterPro.  Use Sheffield&#x2019;s AMBIT Web service to retrieve information about an active site whose characteristics  may be altered due to the presence of a coding SNP which has affected a change in the amino acid sequence  of the protein where the active site is encoded.</li>
</ul>
<p>The <a class="externalLink" href="http://www.myexperiment.org/workflows/28">workflow</a>  for the Graves Disease analysis is published on <a class="externalLink" href="http://www.myexperiment.org/">myExperiment</a>.</p></div>
<div class="section">
<h3><a name="Publications"></a>Publications</h3>
<p><a class="externalLink" href="http://scholar.google.co.uk/scholar?q=Taverna+Graves+Disease">Articles and papers</a>  about the success of Taverna for Graves Disease research.</p></div></div>
                  </div>
          </div>

    <hr/>

    <footer>
            <div class="container">
                      <div class="row">
                                      <p >Copyright &copy;                    2015
                        <a href="http://www.apache.org/">The Apache Software Foundation</a>.
            All rights reserved.      
                    
      </p>
                </div>

                <p id="poweredBy" class="pull-right">
                                                                                                                <a href="http://maven.apache.org/" title="Maven" class="builtBy">
        <img class="builtBy"  alt="Maven" src="http://maven.apache.org/images/logos/maven-feather.png"    />
      </a>
                  </p>
        
                </div>
    </footer>
        </body>
</html>
